Crucial MX500: bogus raw values in attribute 197

This patch reverts the bogus raw value fix.

Not all drives return the bogus value; if yours does, then remove this
patch and let the fix do its thing.

https://www.smartmontools.org/ticket/1294

--- a/drivedb.h	2019-12-28 21:40:37.000000000 +0000
+++ b/drivedb.h	2020-03-13 12:10:21.404173825 +0000
@@ -356,7 +356,7 @@
     "CT(250|500|1000|2000)MX500SSD[14]", // tested with CT500MX500SSD1/M3CR023,
       // .../M3CR020, .../M3CR022
     "M3CR02[0-3]", // Firmware with bogus attribute 197 (see ticket #1227)
-    "This firmware returns bogus raw values in attribute 197",
+    "",
   //"-v 1,raw48,Raw_Read_Error_Rate "
     "-v 5,raw48,Reallocate_NAND_Blk_Cnt "
   //"-v 9,raw24(raw8),Power_On_Hours "
@@ -371,7 +371,7 @@
   //"-v 187,raw48,Reported_Uncorrect "
   //"-v 194,tempminmax,Temperature_Celsius "
   //"-v 196,raw16(raw16),Reallocated_Event_Count "
-    "-v 197,raw48,Bogus_Current_Pend_Sect " // Randomly flips 0 <> 1
+  //"-v 197,raw48,Bogus_Current_Pend_Sect " // Randomly flips 0 <> 1
   //"-v 198,raw48,Offline_Uncorrectable "
   //"-v 199,raw48,UDMA_CRC_Error_Count "
     "-v 202,raw48,Percent_Lifetime_Remain "
